terraform/applications/pastebin/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "cost_center" {
+  default = "1410"
+  type    = string
+}
+
+variable "environment" {
+  default = "prod"
+  type    = string
+}
+
+variable "project" {
+  default = "pastebin"
+  type    = string
+}
+
+variable "project_desc" {
+  default = "paste.allizom.org"
+  type    = string
+}
+
+variable "project_email" {
+  default = "it-sre@mozilla.com"
+  type    = string
+}
+
+variable "region" {
```

Comment:
We redefine `cost_center=1410`, `environment=prod`, `region=us-west-2` here for pastebin, when they have those values already in variables.tf. Duly noted in case that's duplicated by accident rather than by design; or, I may have misunderstand the inheritance link between the two files.

Reply:
@floatingatoll do you mean redefined between variables.tf & variables.tfvars? Its a terraform-ism - e.g. one defines the variable instances, the other defines the variable instance values. However, a lot of web sre current terraform just uses that default field in variables.tf bc we don't do anything too involved with our terraform usage at the moment.